source/developer/interactive-dialogs.rst

```diff
@@ -130,13 +130,15 @@ The list of supported fields is the same as for the ``textarea`` type element.
     "display_name", "String", "Display name of the field shown to the user in the dialog. Maximum 24 characters."
     "name", "String", "Name of the field element used by the integration. Maximum 300 characters. You should use unique “name” fields in the same dialog."
     "type", "String", "Set this value to ``textarea`` for a textarea element."
-    "subtype", "String", "(Optional) One of ``text``, ``email``, ``number``, ``tel``, or ``url``. Default is ``text``. Use this to set which keypad is presented to users on mobile when entering the field."
+    "subtype", "String", "(Optional) One of ``text``, ``email``, ``number``, ``password``,``tel``, or ``url``. Default is ``text``. Use this to set which keypad is presented to users on mobile when entering the field."
```

Comment:
I would like to also see the required Mattermost version for this documented. But I'm unsure how to document this properly. Maybe we add a short section for each `subtype` with a screenshot? 1/5

Reply:
I added the 5.14 requirement in the text - it's on Master already:  https://github.com/mattermost/docs/blob/master/source/developer/interactive-dialogs.rst